docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers **CIFuzz**, which will run your fuzz targets each time a pull request
+is submitted, for projects hosted on GitHub. This allows you to detect and
+fix bugs before they are submitted.
+
+## How it works
+
+CIFuzz works by checking out a repository at the head of a pull request. The
+project's fuzz targets are built and run for
+a definite amount of time (default is 10 minutes). If a bug is found, the
```

Comment:
I'd probably use "crash" instead of the "bug", but not a big deal

Reply:
Done.